scripts/py2dsc

```diff
@@ -7,9 +7,17 @@ where distfile is a .zip or .tar.gz file built with the sdist command
 of distutils.
 """
 
-import stdeb.cli_runner
+import os
 import sys
 
+# allow to run script from source code checkout
+script_path = os.path.abspath(os.path.dirname(__file__))
+if os.path.exists(script_path + '/../setup.py'):
```

Comment:
searching for a `setup.py` is way too flaky for this.  It could occur in the wild, and would be quite confusing for the user.

Overall this patch is unnecessary, and shouldnt be added to runtime systems.  `PYTHONPATH` can be used to get these scripts working without them being installed.  You could have a wrapper script which does this with the first arg being the script name followed by its args.

Reply:
@jayvdb setting up wrapper script or `PYTHONPATH` is no different than installing this tool, which contradicts with the purpose of the patch - to run scrips directly from checkout without any additional documentation or installation steps.